src/cli/commands/force/lightning/local/device/list.ts

```diff
@@ -44,27 +44,11 @@ export default class List extends Setup {
         this.logger.info(`Device List command invoked for ${platform}`);
 
         return new Promise<any>((resolve, reject) => {
-            super
-                .run() // run setup first
-                .then((result) => {
-                    this.logger.info(
-                        'Setup requirements met, continuing with device list'
-                    );
+            const deviceList = CommandLineUtils.platformFlagIsIOS(platform)
+                ? this.iOSDeviceList()
+                : this.androidDeviceList();
 
-                    const deviceList = CommandLineUtils.platformFlagIsIOS(
-                        platform
-                    )
-                        ? this.iOSDeviceList()
-                        : this.androidDeviceList();
-
-                    resolve(deviceList);
-                })
-                .catch((error) => {
-                    this.logger.warn(
-                        `Device list failed for ${platform}. Setup requirements have not been met.`
-                    );
-                    reject(error);
-                });
+            resolve(deviceList);
```

Comment:
This is fine for now, but based on our discussion I think we want to move the error output generated from the adverse case to debug logging, and simply go with no error message to the console?  Or would you all prefer some sort of error message to the console in the case where we can't generate a list?  @trooper2013 @brandonpage thoughts?

Reply:
(I commented here, but the place for the change is probably `AndroidSDKUtils.fetchEmulators()` and `IOSUtils.getSupportedSimulators()`?)

Reply:
My $0.02 is that if the error messages turn out to be as benign on iOS as they are on Android, we could probably just leave them in as is.

![image](https://user-images.githubusercontent.com/755462/86836738-c6992d00-c052-11ea-9f68-7606e8dca7a4.png)

Reply:
I think we'll be getting a similar benign error on iOS too since it is just trying to run `/usr/bin/xcrun simctl list --json devices available` and so it should give a similar `no such file or directory` error if xcode is not installed.